force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -182,15 +213,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return A List of integers containing the numbers 1, 2, and 3.
      */
     public static List<Integer> createAndPopulateList() {
-        // Create a new list of integers
-
-        // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
-
-        // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        List<Integer> my_list = new List<Integer>();
+            my_list.add(1);
+            my_list.add(2);
+            my_list.add(3);
+        return my_list; // Replace null with the variable you used to store the result
```

Comment:
Good work. A pro tip here is you can use the List class Constructor if you know your values ahead of time. It save a few lines and it is a little more performant. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
 ```